Plugins/IMA/IMAPlugin.swift

```diff
@@ -265,16 +267,20 @@ enum IMAState: Int, StateProtocol {
         // Ad break, will be called before each scheduled ad break. Ad breaks may contain more than 1 ad.
         // `event.ad` is not available at this point do not use it here.
         case .AD_BREAK_READY:
-            self.notify(event: AdEvent.AdBreakReady())
-            guard canPlayAd(forState: currentState) else { return }
-            self.start(adsManager: adsManager)
+            if shouldDiscardAd() {
+                PKLog.debug("discard Ad Break")
```

Comment:
We can't discardAdBreak in adBreakReady because no ad have been loaded yet, when we don't call .start() on ad break it is like discarding, if we will call discard here it will discard the next adBreak which is unwanted behavior.